src/components/FilterGroup.tsx

```diff
@@ -104,7 +100,8 @@ export function FilterGroup({
             />
           );
         })}
-        {limited && <button className='text-blue-500 py-1 text-sm' onClick={toggleShowAll}>{showAll ? 'Show Less' : 'Show More'}</button>}
+        {/* eslint-disable-next-line react-perf/jsx-no-new-function-as-prop */}
+        {limited && <button className='text-primary py-1 text-sm' onClick={_e => setShowAll(!showAll)}>{showAll ? 'Show Less' : 'Show More'}</button>}
```

Comment:
Can this be multiple lines? You can also replace the `_e` with `()`

Reply:
not sure if I formatted it right lol, could you take a look?